pkg/daemon/multus/config.go

```diff
@@ -38,6 +38,8 @@ var (
 var (
 	DefaultValidationNamespace = "rook-ceph"
 
+	DefaultServiceAccountName = ""
```

Comment:
How about using the `k8sutil.DefaultServiceAccount` to set `rook-ceph-default`? Or does the default need to be blank?

Reply:
Let me think about this more and do some testing on minikube and OpenShift. This could be a good way to avoid having to use the `multus-validation-test-openshift.yaml` RBAC resources (or equivalent configuration on other security-constrained K8s distros), but I should verify to be sure.

Reply:
I'm trying to use the `rook-ceph-system` service account on openshift (from operator hub), and it isn't working again just as here: https://github.com/rook/rook/pull/14230#issuecomment-2155664191

```
Warning  FailedCreate  10s (x12 over 20s)  daemonset-controller  Error creating: pods "multus-validation-test-host-checker-shared-storage-and-worker-nodes-" is forbidden: unable to validate against any security context constraint: [provider "anyuid": Forbidden: not usable by user or serviceaccount, provider restricted-v2: .spec.securityContext.hostNetwork: Invalid value: true: Host network is not allowed to be used, provider restricted-v2: .containers[0].hostNetwork: Invalid value: true: Host network is not allowed to be used, provider "restricted": Forbidden: not usable by user or serviceaccount, provider "nonroot-v2": Forbidden: not usable by user or serviceaccount, provider "nonroot": Forbidden: not usable by user or serviceaccount, provider "noobaa": Forbidden: not usable by user or serviceaccount, provider "hostmount-anyuid": Forbidden: not usable by user or serviceaccount, provider "machine-api-termination-handler": Forbidden: not usable by user or serviceaccount, provider "hostnetwork-v2": Forbidden: not usable by user or serviceaccount, provider "hostnetwork": Forbidden: not usable by user or serviceaccount, provider "hostaccess": Forbidden: not usable by user or serviceaccount, pod.metadata.annotations[seccomp.security.alpha.kubernetes.io/pod]: Forbidden: seccomp may not be set, pod.metadata.annotations[container.seccomp.security.alpha.kubernetes.io/readiness-check-web-server-public-addr]: Forbidden: seccomp may not be set, provider "node-exporter": Forbidden: not usable by user or serviceaccount, provider "rook-ceph-csi": Forbidden: not usable by user or serviceaccount, provider "privileged": Forbidden: not usable by user or serviceaccount]
```

I feel like there must be something about the pods being created by the validation tool is stopping them from being able to see the `rook-ceph` SCC that is installed there.

From the SCC itself, everything looks like it should be fine.
```
oc describe scc rook-ceph
Name:						rook-ceph
Priority:					<none>
Access:
  Users:					system:serviceaccount:openshift-storage:rook-ceph-system,system:serviceaccount:openshift-storage:default,system:serviceaccount:openshift-storage:rook-ceph-mgr,system:serviceaccount:openshift-storage:rook-ceph-osd,system:serviceaccount:openshift-storage:rook-ceph-rgw
  Groups:					<none>
Settings:
  Allow Privileged:				true
  Allow Privilege Escalation:			true
  Default Add Capabilities:			<none>
  Required Drop Capabilities:			ALL
  Allowed Capabilities:				MKNOD
  Allowed Seccomp Profiles:			<none>
  Allowed Volume Types:				configMap,downwardAPI,emptyDir,hostPath,persistentVolumeClaim,projected,secret
  Allowed Flexvolumes:				<all>
  Allowed Unsafe Sysctls:			<none>
  Forbidden Sysctls:				<none>
  Allow Host Network:				true
  Allow Host Ports:				true
  Allow Host PID:				false
  Allow Host IPC:				true
  Read Only Root Filesystem:			false
  Run As User Strategy: RunAsAny
    UID:					<none>
    UID Range Min:				<none>
    UID Range Max:				<none>
  SELinux Context Strategy: MustRunAs
    User:					<none>
    Role:					<none>
    Type:					<none>
    Level:					<none>
  FSGroup Strategy: MustRunAs
    Ranges:					<none>
  Supplemental Groups Strategy: RunAsAny
    Ranges:					<none>
```